Doc/reference/datamodel.rst

```diff
@@ -1946,10 +1945,11 @@ Once the appropriate metaclass has been identified, then the class namespace
 is prepared. If the metaclass has a ``__prepare__`` attribute, it is called
 as ``namespace = metaclass.__prepare__(name, bases, **kwds)`` (where the
 additional keyword arguments, if any, come from the class definition). The
-``__prepare__`` method should be implemented as a :func:`classmethod`.
++namespace returned by ``__prepare__`` is passed in to ``__new__``, but when
++the final class object is created the namespace is copied into a new ``dict``.
 
 If the metaclass has no ``__prepare__`` attribute, then the class namespace
-is initialised as an empty ordered mapping.
+ is initialised as an empty :func:`dict` instance.
```

Comment:
No leading space here. Also, no need to add "instance":

```suggestion
is initialised as an empty :func:`dict`.
```

Separately, I have no idea if this is guaranteed by the language to be a `dict`. Before, it wasn't, but with this new change it certainly is.

If that's the case, then I'm against changing this line.